### PR TITLE
Added instructions for branching

### PR DESCRIPTION
**Pull request by gautamkrishnar**
Made changes to cope with PR #326 
So that new contributors can easily use branches.

### CODE REVIEW

File: duckduckhack/submitting-your-instant-answer/submission_and_review.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ Here is how to commit your code in the Codio environment:
     To https://github.com/GitHubUsername/zeroclickinfo-goodies.git
        138f5bc..c69c517  master -> master
     ```
+If you had created a bramch for your current contribution then you need to Type **`git push origin branch_name`** and press **`Enter`**. Then you need to type your username and password.
```
Comment (moollaza) [inline duckduckhack/submitting-your-instant-answer/submission_and_review.md]: typo: "create a **branch**"

Comment (moollaza) [inline duckduckhack/submitting-your-instant-answer/submission_and_review.md]: I don't *think* they need to enter their branch name. "git push origin" should push to a new branch on the remote with the same name as the current, but explicitly providing the name doesn't hurt.

**Comment by moollaza** [comment]
Once the typo is fixed, I'm OK with this. Passing on to @talsraviv to review + merge.

**Comment by gautamkrishnar** [comment]
@moollaza :+1:

**Comment by gautamkrishnar** [comment]
@talsraviv have you forgotten about this? :smile:

File: duckduckhack/submitting-your-instant-answer/submission_and_review.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ Here is how to commit your code in the Codio environment:
     To https://github.com/GitHubUsername/zeroclickinfo-goodies.git
        138f5bc..c69c517  master -> master
     ```
+If you had created a branch for your current contribution then you need to Type **`git push origin branch_name`** and press **`Enter`**. Then you need to type your username and password.
```
Comment (moollaza) [inline duckduckhack/submitting-your-instant-answer/submission_and_review.md]: Let's move this above so we don't have them read the wrong instructions first.

Also, let's rephrase slightly:

"If you created a branch for your current contribution, then type **`git push origin branch_name`** and press **`Enter`** If not, type **`git push**` and press **`Enter`**. Git will print some text ..."

**Comment by talsraviv** [comment]
@gautamkrishnar my bad! I didn't see the mention last week. Sorry about that. Agree with @moollaza

**Comment by gautamkrishnar** [comment]
@talsraviv is everything okay now!! ?

**Comment by talsraviv** [comment]
:+1: Looks good, merging

**Comment by gautamkrishnar** [comment]
Thanks @moollaza @talsraviv  :smile:  :+1:

**Pull request merged**